gui/kitchen-info.lua

```diff
@@ -0,0 +1,414 @@
+-- Adds more info to the Kitchen screen.
+--@ enable = true
+--[====[
+
+gui/kitchen-info
+===================
+Adds more info to the Kitchen screen.
+
+Usage::
+
+    gui/kitchen-info enable|disable|help
+    enable|disable gui/kitchen-info
+
+]====]
+
+gui = require 'gui'
+dialogs = require 'gui.dialogs'
+gps = df.global.gps
+world = df.global.world
+
+local kitchen_overlay = defclass(kitchen_overlay, gui.Screen)
```

Comment:
in order for class identity to be preserved across script invocations, classes cannot be declared `local`

Reply:
Fixed